phantombot/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
```

Comment:
Umbrel's app proxy service should be added here as the first service, which can then point to the `phantombot_server_1` container and default port `25000`. That way you can remove the port mapping in your `server` service and then PhantomBot will running behind our transparent proxy and inherit the security properties of our umbrel auth:

firefox example to follow
https://github.com/getumbrel/umbrel-apps/blob/2dbb41875664e250d34211d395523e46550b5def/firefox/docker-compose.yml#L3-L7

- By default, we only let requests through the proxy if the user already has a valid auth cookie from the Umbrel homescreen. So if the PhantomBot dashboard has no auth of its own, it will still be protected by ours. This gives a good UX because there's zero friction for Umbrel users... they can just access the dashboard without re-entering any credentials if they are already logged in to their Umbrel.
- It also has the benefit of inheriting other security properties of Umbrel auth, such as 2FA if they have it enabled on their Umbrel. They would then get 2FA security for all apps behind the auth proxy too.
- And then if you need to allow external connections to PhantomBot running on port 25000 (e.g., not to the UI but to the api or something), you can whitelist certain routes, so that the web UI is protected by auth, but something like /api/* isn't:

e.g., adding this env var to the proxy container
```yaml
PROXY_AUTH_WHITELIST: "/api/*"
```

- You can also disable the auth portion of the app proxy container entirely with this env var:
```yaml
PROXY_AUTH_ADD: "false"
```